Maestro/subscriptions.json

```diff
@@ -41,6 +41,22 @@
     {
       "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt",
       "handlers": [
+        // This handler will bring the Latest CoreFX release/1.0.0 build into CoreFX release/1.0.0
```

Comment:
Correct we try to keep the corefx repo itself building against the latest packages that corefx produces.